site/content@beta/backend-apis/apis.md

```diff
@@ -0,0 +1,276 @@
+---
+weight: 40
+---
+
+# Agent Chat API
+
+## Use cases
+
+The **Agent Chat API** allows for: 
+
+- interaction with the chat as Agent (joining a chat, posting messages)
+- interact with the chat by Bot Agents (acting as Agents) 
+- building a custom Agent App
+- browsing chat archives
+- banning Customers 
+- and much more
+
+If you plan on using the **Agent Chat API** as **RTM API**, refer to [Agent Chat RTM API](../agent-chat-rtm-api). For the **Web API** usage, read the [Agent Chat Web API](../agent-chat-web-api) document.
+
+## Chatting as Agent
+
+The number of chats an Agent can participate in is not limited. It can be configured in the Agent App by the licence Owner or Admin.
+
+# Customer Chat API
+
+## Use cases
+
+The [**Customer Chat API**](../customer-chat-api) allows for:
+
+- interacting with the chat as Customer (join a chat, post messages)
+- building a custom Chat Widget
+- implementing new conversation channels (Facebook Messenger or Twitter)
+
+<!-- If you plan on using the **Customer Chat API** as **RTM API**, refer to [Customer Chat RTM API](../customer-chat-rtm-api). For the **Web API** usage, read the [Customer Chat Web API](../customer-chat-web-api) document. -->
```

Comment:
Once we split the Customer Chat API document in the RTM and Web versions, this description will be valid.
For now, we can mention we're working on this doc.